cmd/buildctl/main.go

```diff
@@ -87,6 +87,10 @@ func main() {
 			Usage: "timeout backend connection after value seconds",
 			Value: 5,
 		},
+		cli.BoolFlag{
+			Name:  "wait",
```

Comment:
This isn't consistent with the flag mentioned in the PR title

Reply:
Personally, I prefer the shorter form `wait` - I'll update the PR title to match.

Happy to discuss though, I don't have strong opinions.